cirq/google/programs_test.py

```diff
@@ -379,10 +381,12 @@ def test_w_to_proto_dict():
             }
         }
     }
-    assert_proto_dict_convert(gate, proto_dict,
-                              cirq.GridQubit(2, 3))
+    with pytest.raises(NotImplementedError):
```

Comment:
This should still succeed. Do not expect a failure here.

Reply:
Done. And all the other ones too.